okapi-core/src/main/java/org/folio/okapi/managers/TenantManager.java

```diff
@@ -1044,6 +1085,68 @@ private ModuleInstance getTenantInstanceForInterface(
     });
   }
 
+  /**
+   * Get module cache for tenant.
+   * @param tenant Tenant
+   * @return Module Cache
+   */
+  public Future<ModuleCache> getModuleCache(Tenant tenant) {
+    if (!enabledModulesCache.containsKey(tenant.getId())) {
+      return Future.succeededFuture(new ModuleCache(new LinkedList<>()));
+    }
+    return Future.succeededFuture(enabledModulesCache.get(tenant.getId()));
+  }
```

Comment:
This should be replaced by 

`return Future.succeededFuture(enabledModulesCache.computeIfAbsent(tenant.getId(), key -> new ModuleCache(new LinkedList<>())));`

Reply:
More elegant indeed.